common/util/BUILD

```diff
@@ -134,13 +136,18 @@ cc_library(
     ],
 )
 
+# TODO: once all absl logging features are established in abseil-cpp, we
+# should IWYU them directly in places where we need logging and remove
+# this common/util:logging target.
 cc_library(
     name = "logging",
+    srcs = ["logging.cc"],
     hdrs = ["logging.h"],
     deps = [
-        "@com_github_google_glog//:glog",
-        "@com_google_absl//absl/base",
         "@com_google_absl//absl/base:core_headers",
+        "@com_google_absl//absl/log:check",
+        "@com_google_absl//absl/log:die_if_null",
+        "@com_google_absl//absl/random",
```

Comment:
Why do we need random now?

Reply:
Good question, looks like a copy/paste glitch. Let me remove it and see.